examples/sysl.wbnf

```diff
@@ -45,18 +56,76 @@ view_params -> (NAME "<:" (collection_type|type_spec)):",";
 
 expr_block -> "";
 
+// --------------  statements  --------------- //
+statements -> (if_stmt
+                | for_stmt
+                | ret_stmt
+                | call_stmt
+                | one_of_stmt
+                | http_method_comment=(COMMENT)
+                | group_stmt
+                | text_stmt
+                | annotation
+        ) attribs?;
+
+if_else -> if_stmt else_stmt*;
+
+if_stmt -> "if" PREDICATE ":" INDENT statements*:INDENT_SEP;
+
+else_stmt -> "else" PREDICATE? ":" INDENT statements*:INDENT_SEP;
+
+for_stmt -> for_type=("alt" | "until" | "for each" | "for" | "loop" | "while") PREDICATE ":" INDENT statements*:INDENT_SEP;
+
+one_of_stmt -> "one" "of" ":" INDENT one_of_cases+:INDENT_SEP;
+
+one_of_cases -> one_of_case_label=(NAME | TEXT_LINE | QSTRING)? ":" INDENT statements+:INDENT_SEP;
```

Comment:
Is the `+` required? Ditto elsewhere.

Reply:
just realized it's wrong to use that, will change all of them